docs/src/pages/components/icons/icons.md

```diff
@@ -107,16 +107,31 @@ function HomeIcon(props) {
 You can use the `SvgIcon` wrapper even if your icons are saved the `.svg` format.
```

Comment:
Let's fix this while we're here:

```suggestion
You can use the `SvgIcon` wrapper even if your icons are saved in the `.svg` format.
```

However I'd argue that `.svg` is a file extension, not a format, so:

```suggestion
You can use the `SvgIcon` wrapper even if your icons are saved as SVG files.
```